### PR TITLE
Add ServiceNow integration docs

### PR DESCRIPTION
**Pull request by amirmuminovic**

### CODE REVIEW

**Review by mbonaci — APPROVED** [review]
LGTM

File: docs/integration/alerts-servicenow-integration.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+**2.** Create a new Scripted REST API by clicking **New** on the top bar above the table.
+
+<img class="content-modal-image" alt="Create Service Now Integration - Scripted REST APIs" src="../../images/integrations/create-servicenow-integration-scripted-apis.png" title="Create Service Now Integration - Scripted REST APIs">
+
+**3.** Fill out the name and API ID and then click **Submit**.
+
+<img class="content-modal-image" alt="Create Service Now Integration - Scripted REST API Creation" src="../../images/integrations/create-servicenow-integration-scripted-api-creation.png" title="Create Service Now Integration - Scripted REST API Creation">
+
+**4.** Once you've created the API, you will be taken to the initial Scripted Rest API screen. You can find your application by entering the app name in the search bar and clicking Enter on your keyboard. The app should be shown in the results. Click on the name of the application to continue the setup.
```
Comment (otisg) [inline docs/integration/alerts-servicenow-integration.md:18]: is "app name" referring to "App" in Sematext?

**Review by otisg — APPROVED** [review]
LGTM

File: docs/integration/alerts-servicenow-integration.md (hunk 1 above)
Comment (amirmuminovic) [reply docs/integration/alerts-servicenow-integration.md:18]: Yes

Comment (amirmuminovic) [reply docs/integration/alerts-servicenow-integration.md:18]: Nope, it's the app from Service now or API to be more exact, I'll make the propper adjustments

**Pull request merged**